kubetest/e2e.go

```diff
@@ -762,18 +762,18 @@ func kubemarkGinkgoTest(testArgs []string, dump string) error {
 }
 
 // Brings down the kubemark cluster.
-func kubemarkDown(err *error, wg *sync.WaitGroup, provider, dump string) {
+func kubemarkDown(err *error, logexporterGCSPath, wg *sync.WaitGroup, provider, dump string) {
 	defer wg.Done()
 	control.XMLWrap(&suite, "Kubemark MasterLogDump", func() error {
 		logDumpPath := logDumpPath(provider)
-		cmd := exec.Command(logDumpPath, dump)
+		cmd := exec.Command(logDumpPath, dump, logexporterGCSPath)
```

Comment:
logexporterGCSPath can be empty, in that case it shouldn't be added to exec.Command, like in: https://github.com/kubernetes/test-infra/blob/ee1e7c845bb25177944bb5de37545f007f529059/kubetest/e2e.go#L551

Reply:
Just confirming. There should be an if else condition, one when logexporterGCSPath is present and second when it does not.

Reply:
Yes, this is correct. Thanks for help! :)